src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackV2VolumePluginTest.java

```diff
@@ -84,53 +81,73 @@ public void setUp() throws Exception {
         this.tokenDefault = new Token(FAKE_ACCESS_ID, tokenUser, new Date(), attributes);
     }
```

Comment:
private final String TENANT_ID = "tenantId" in line 42 could be made a local variable (in setUp method, i guess)

Reply:
as well as private HttpRequestClientUtil httpRequestClientUtil in line 59

Reply:
Okay, done!